template/docker/README.md

```diff
@@ -1,2 +1,7 @@
 # Docker build directory
-TODO: write doc
\ No newline at end of file
+Put your Dockerfile and other files relating to a docker-build here. 
+### Building docker image locally
+`make build-docker [IMAGE_NAME=<name>:<tag>]` will look for a Dockerfile in [docker/](docker/) and build an image tagged with `IMAGE_NAME`. If `IMAGE_NAME` is not supplied it will be tagged `template:latest`.
```

Comment:
```suggestion
If you have docker installed on your machine, you may `cd docker; docker build -t my_image:local .`  and build the image.
```

Reply:
This is if we remove `make build-docker` I assume? I like it more, let's do that